com.adilhanney.saber.json

```diff
@@ -0,0 +1,105 @@
+{
+	"app-id": "com.adilhanney.saber",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "22.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "saber",
+	"separate-locales": false,
+	"finish-args": [
+		"--share=network",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland",
+		"--device=dri",
+		"--socket=pulseaudio",
+		"--filesystem=xdg-documents/Saber/:create"
+	],
+	"modules": [
+		{
+			"name": "saber",
+			"buildsystem": "simple",
+			"only-arches": [
+				"x86_64"
+			],
+			"build-commands": [
+				"mkdir -p Saber",
+				"tar -xf Saber-Linux-Portable.tar.gz -C Saber",
+
+				"cp -r Saber /app/",
+				"chmod +x /app/Saber/saber",
+				"mkdir -p /app/bin",
+				"ln -s /app/Saber/saber /app/bin/saber",
+
+				"install -Dm644 com.adilhanney.saber.metainfo.xml -t /app/share/metainfo/",
+
+				"install -Dm644 icon-16x16.png /app/share/icons/hicolor/16x16/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-24x24.png /app/share/icons/hicolor/24x24/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-32x32.png /app/share/icons/hicolor/32x32/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-48x48.png /app/share/icons/hicolor/48x48/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-128x128.png /app/share/icons/hicolor/128x128/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-256x256.png /app/share/icons/hicolor/256x256/apps/com.adilhanney.saber.png",
+				"install -Dm644 icon-512x512.png /app/share/icons/hicolor/512x512/apps/com.adilhanney.saber.png",
+
+				"install -Dm644 com.adilhanney.saber.desktop -t /app/share/applications/"
+			],
+			"sources": [
```

Comment:
Nitpick: I don't think I understand why icons, metainfo and desktop files are not part of the `tar.gz`.

Reply:
> Nitpick: I don't think I understand why icons, metainfo and desktop files are not part of the `tar.gz`.


The `tar.gz` file is just what Flutter outputs with the build command. I think they probably don't include icons since their distribution method of choice is Snap. Including the icons this way seems easier than repacking the build archive